### PR TITLE
Add codepen to examples - part 2!

### PR DESCRIPTION
**Pull request by spectranaut**
This PR adds codepen buttons to the following examples! To test locally, you will have to serve the example file with a web server ([described here](https://github.com/w3c/aria-practices/wiki/How-to-add-the-Open-In-Codepen-button-to-an-example-page#how-to-test-your-changes-locally)).

### RawGitHack preview links

- [tabs/tabs-1/tabs.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/tabs/tabs-1/tabs.html)
- [tabs/tabs-2/tabs.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/tabs/tabs-2/tabs.html)
- [checkbox/checkbox-1/checkbox-1.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/checkbox/checkbox-1/checkbox-1.html)
- [checkbox/checkbox-2/checkbox-2.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/checkbox/checkbox-2/checkbox-2.html)
- [menu-button/menu-button-actions.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/menu-button/menu-button-actions.html)
- [menu-button/menu-button-actions-active-descendant.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part2/examples/menu-button/menu-button-actions-active-descendant.html)

### CODE REVIEW

**Comment by spectranaut** [comment]
I made this PR on top of https://github.com/w3c/aria-practices/pull/1532, which I expect will land soon!

**Review by carmacleod — APPROVED** [review]
Looks good, works great!
+1

File: examples/menu-button/menu-button-actions-active-descendant.html [hunk 1]
```diff
@@ -321,9 +323,9 @@ <h3 id="rps2_label">Menu</h3>
       <section>
         <h2>Javascript and CSS Source Code</h2>
 
-        <ul>
+        <ul id="css_js_files">
           <li>CSS: <a href="css/menuButtonAction.css" type="text/css">MenubuttonAction.css</a></li>
-          <li>Javascript: <a href="js/Menubutton2.js" type="text/javascript">Menubutton2.js</a></li>
+          <li>Javascript: <a href="js/Menubutton.js" type="text/javascript">Menubutton2.js</a></li>
```
Comment (carmacleod) [inline examples/menu-button/menu-button-actions-active-descendant.html:328]: Should change the link text as well, to match the href.

```suggestion
          <li>Javascript: <a href="js/Menubutton.js" type="text/javascript">Menubutton.js</a></li>
```

**Comment by carmacleod** [comment]
+1 for the 6 more examples!

**Pull request merged**